src/tests/CMakeLists.txt

```diff
@@ -55,3 +57,7 @@ foreach(PROG
     set_tests_properties(memory_${PROG} PROPERTIES LABELS "tools;votca;memory")
   endif(VALGRIND_FOUND) 
 endforeach(PROG)
+
+
+execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/DataFiles ${CMAKE_BINARY_DIR}/tools/src/tests/DataFiles)
```

Comment:
I don't get it, if you symlink the whole folder instead of the individual files, you can as well just not do it. As files that get written to the symlink folder will end up in the source folder. Just do `TOOLS_TEST_FOLDER="${CMAKE_CURRENT_SOURCE_DIR}`.

Reply:
I do not understand. The folder is not suposed to be written to.

Reply:
Nobody is supposed to write to that folder. Nobody is prevented from writing to source anyway. I agree it is not brilliant but symlinking individual files is really cumbersome.

Reply:
Okay I have a fix I hope you like it